<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover {color : rgb(123, 151, 240)}
QPushButton:pressed {color : rgb(77, 110, 200)}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>941</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>36</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sleep signal demonstrator</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbMenu">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>401</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnOpenPatientDialog">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>300</width>
       <height>51</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolTip">
      <string extracomment="Register or select a patient"/>
     </property>
     <property name="text">
      <string>Select patient</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenDeviceDialog">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolTip">
      <string extracomment="Select a device of choice"/>
     </property>
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenRecordingDialog">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolTip">
      <string extracomment="Start a recording"/>
     </property>
     <property name="text">
      <string>Recording</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenAnalysisDialog">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolTip">
      <string extracomment="Start a analysis"/>
     </property>
     <property name="text">
      <string>Analysis</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnOpenExportDialog">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>300</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="toolTip">
      <string extracomment="Manage the system"/>
     </property>
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbInfoPatient">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>341</width>
      <height>271</height>
     </rect>
    </property>
    <property name="toolTip">
     <string extracomment="Information about the patient"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>301</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblNamePatient">
        <property name="text">
         <string>name patient...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblEmail">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Gender:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblGenderPatient">
        <property name="text">
         <string>o</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>BirthDate:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblBirthDatePatient">
        <property name="text">
         <string>12-12-2012</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Phone:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="lblPhone">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="lblHeight">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Weight:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="lblWeight">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>BMI:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLabel" name="lblBMI">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Information patient</string>
     </property>
    </widget>
    <widget class="Line" name="lineInfoPatient">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSelectDirPatient">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Directory patient</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddNotes">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Add notes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnChangePatient">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>220</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Change patient</string>
     </property>
    </widget>
    <widget class="Line" name="lineInfoPatient_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>971</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
